bin/codemods/src/single-tree-rendering.js

```diff
@@ -256,6 +295,33 @@ export default function transformer( file, api ) {
 		.replaceWith( ensureNextMiddleware )
 		.forEach( transformReactDomRender );
 
+	// Find if `reactDom` is used
+	const reactDomDefs = root.find( j.MemberExpression, {
+		object: {
+			name: 'ReactDom',
+		},
+	} );
+
+	// Remove stranded `reactDom` imports
```

Comment:
Nitpick: Should be either `ReactDom` or `react-dom` in the comment. (I kinda stumble over it when reading, might be my OCD 😄)